Bump version and update changelog (#4028)
SEED-platform/seed: CHANGELOG.md

# SEED Version 2.18.1
<!-- Release notes generated using configuration in .github/release.yml at develop -->

## What's Changed
### New Features 🎉
* Codebuild & Elasticache Redis integration support by @dhaley in https://github.com/SEED-platform/seed/pull/3981
### Improvements 📈

> * add better error messaging that includes the property view ID by @kflemin in https://github.com/SEED-platform/seed/pull/3973

Reviewer: ```suggestion
* Add better error messaging that includes the property view ID by @kflemin in https://github.com/SEED-platform/seed/pull/3973
```

* Add Salesforce fields to store data admin contact by @kflemin in https://github.com/SEED-platform/seed/pull/3985
* Add salesforce mappings sort and more error checking and messaging by @kflemin in https://github.com/SEED-platform/seed/pull/3992
* Add zoom and panning to insights property page by @haneslinger in https://github.com/SEED-platform/seed/pull/3879
* Remove program overview tabular data by @haneslinger in https://github.com/SEED-platform/seed/pull/4007
* Save Insight property config locally by @haneslinger in https://github.com/SEED-platform/seed/pull/3994

> * adding sf delete_label_on_sync setting by @kflemin in https://github.com/SEED-platform/seed/pull/4022

Reviewer: ```suggestion
* Add Salesforce delete_label_on_sync setting by @kflemin in https://github.com/SEED-platform/seed/pull/4022
```

> * replacing seed getting started guide for v2.18 by @kflemin in https://github.com/SEED-platform/seed/pull/4026

Reviewer: ```suggestion
* Replace seed getting started guide for v2.18 by @kflemin in https://github.com/SEED-platform/seed/pull/4026
```
